weed/shell/shell_liner.go

```diff
@@ -84,6 +92,8 @@ func RunShell(options ShellOptions) {
 		}
 
 		for _, c := range util.StringSplit(cmd, ";") {
+			glog.V(0).Infof("%s%s", "> ", cmd)
+			fmt.Printf("%s%s\n", "> ", cmd)
```

Comment:
these 2 lines are the same?

Reply:
repeat:
glog & fmt, glog goes into stderr and fmt does stdout.

Only the fmt line goes into file when piped as output ` > out.txt`

Maybe there's a better way to do it but I didn't see glog & stdout. just stderr and reference to why people have made other logger libraries.